blog.md

```diff
@@ -0,0 +1,94 @@
+# Scraping and Sanitizing Data in Python by Beautiful Soup
```

Comment:
nit: can consider another title, it's a little clunky as is but I like having all of "Scraping, Python, and Beautiful Soup" in there